addons/account/tests/test_account_move_in_invoice.py

```diff
@@ -2124,3 +2129,38 @@ def test_invoice_sent_to_additional_partner(self):
         self.assertTrue(follower_mail)
         self.assertNotIn('access_token=', follower_mail.body_html,
                       "The followers should not bet sent the access token by default")
+
+    def test_purchase_uom_on_vendor_bills(self):
+        def create_move_and_return_uom(move_type, product):
+            move = self.env['account.move'].create({
+                'move_type': move_type,
+                'partner_id': self.partner_a.id,
+                'invoice_date': fields.Date.from_string('2019-01-01'),
+                'invoice_payment_term_id': self.pay_terms_a.id,
+                'invoice_line_ids': [
+                    Command.create({}),
+                ],
```

Comment:
not needed if you do `with move_form.invoice_line_ids.new() as move_line:` later :)